### ISSUE TITLE
Option to show stdout in messages list

### ISSUE THREAD

**Issue by notitan** [xgsa/cdt-tests-runner#14]
When a Boost.Test runs with the --output_format=XML option, it puts the stdout messages (like written with a std::cout command) to a node named <TestLog>. This node can not be viewed in the Eclipse view of cdt-test-runner and is also not shown in the Eclipse Console view.
It would be great to have an option beside error, warning, information to show the console output as well.

**Comment by xgsa** [comment]
Agree, it would be nice to have such feature, I'm missing it too. Unfortunately currently I have no time to implement it, but possibly it will appear some day in future.

**Comment by notitan** [comment]
I have no experience with eclipse plugins, but I will have a look and maybe I can help.
